src/routes/safe/container/index.tsx

```diff
@@ -40,7 +42,7 @@ const Container = (): React.ReactElement => {
     onClose: () => {},
   })
 
-  if (!featuresEnabled) {
```

Comment:
`featuresEnabled` is always truthy because it's an array. I've opted for checking for the number of owners instead.